app/server/db/index.ts

```diff
@@ -14,7 +14,7 @@ export const getDatabaseUrl = () => {
 
   databaseURL += PGUSER;
   if (process.env.PGPASSWORD) {
-    databaseURL += `:${process.env.PGPASSWORD}`;
+    databaseURL += `:${encodeURIComponent(process.env.PGPASSWORD)}`;
```

Comment:
the passwords generated by the postgres operator may have special characters in them

Reply:
weren't we able to pull that URL from the secret generated by the operator? that might be a tech debt item to keep it compatible with local dev